client/webpack.config.js

```diff
@@ -221,12 +223,15 @@ const webpackConfig = {
 			} ),
 			SassConfig.loader( {
 				includePaths: [ __dirname ],
-				postCssConfig: {
-					path: __dirname,
-					ctx: {
-						transformCssProperties: browserslistEnv === 'defaults',
-						customProperties: calypsoColorSchemes,
-					},
+				postCssOptions: {
+					plugins: () =>
+						[
+							autoprefixerPlugin(),
+							browserslistEnv === 'defaults' &&
+								postcssCustomPropertiesPlugin( {
+									importFrom: [ calypsoColorSchemes ],
+								} ),
+						].filter( Boolean ),
```

Comment:
Is there a reason to making plugins a function instead of a plain array? I might be mistaken but PostCSS API [seems to accept an array](https://github.com/postcss/postcss#options).

Reply:
Not really, it can be a plain array just fine 🙂 All the variables that are used to compute the plugin list are global constants. Fixed in https://github.com/Automattic/wp-calypso/pull/45030/commits/f42a554dbabb2429f7309dfedb1f7d2090ab0eb5